receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -173,3 +175,40 @@ func (ghs *githubScraper) getContributorCount(
 
 	return len(all), nil
 }
+
+// Get the pull request data from the GraphQL API.
+func (ghs *githubScraper) getPullRequests(
+	ctx context.Context,
+	client graphql.Client,
+	repoName string,
+) ([]PullRequestNode, error) {
+	var prCursor *string
+	var pullRequests []PullRequestNode
+
+	for hasNextPage := true; hasNextPage; {
+		prs, err := getPullRequestData(
+			ctx,
+			client,
+			repoName,
+			ghs.cfg.GitHubOrg,
+			100,
+			prCursor,
+			[]PullRequestState{"OPEN", "MERGED"},
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		pullRequests = append(pullRequests, prs.Repository.PullRequests.Nodes...)
+		prCursor = &prs.Repository.PullRequests.PageInfo.EndCursor
+		hasNextPage = prs.Repository.PullRequests.PageInfo.HasNextPage
+	}
+
+	return pullRequests, nil
+}
+
+// Get the age/duration between two times in hours. The hours will be rounded up
```

Comment:
I'm thinking that it'd be better to convert to hours at the visualization/query level and the time/age value emitted from the receiver should be in seconds. The semantic conventions seem to leave it open in the case of time for metrics. 

> time - an instrument that measures passage of time should be called entity.time. For example, system.cpu.time with attribute state = idle | user | system | .... time measurements are not necessarily wall time and can be less than or greater than the real wall time between measurements.
> time instruments are a special case of usage metrics, where the limit can usually be calculated as the sum of time over all attribute values. utilization for time instruments can be derived automatically using metric event timestamps. For example, system.cpu.utilization is defined as the difference in system.cpu.time measurements divided by the elapsed time and number of CPUs.

This is primarily focused at time in the context of system metrics. I think just recording the real time in s/ms would be fine. Wouldn't mind knowing your thoughts on this @atoulme / @astencel-sumo

Reply:
http semconv just forced request duration to seconds from ms, so it seems like using seconds is the right answer.

Reply:
The latest commits address this and we now emit seconds instead of hours.